tests/test_doc2vec.py

```diff
@@ -23,7 +23,7 @@ def det_hash(x):
 class Doc2VecTest(BaseTest):
 
     def test_fit_transform(self):
-        tw = TextWiser(Embedding.Doc2Vec(seed=1234, vector_size=2, min_count=1, workers=1, sample=0, negative=0,
+        tw = TextWiser(Embedding.Doc2Vec(seed=1234, vector_size=2, min_count=1, workers=1, sample=0, negative=5,
```

Comment:
Yes, we can do that. I believe we already have a validate function for the Doc2Vec class and we can extend its functionality to check for  unsupported keyword argument values as well!

Reply:
We actually don't do any validation of the arguments passed to Doc2Vec itself. We can of course add this one, but to validate all the arguments for Doc2Vec (and similarly other embedding/transformation methods) would be substantial effort.

Reply:
@bkleyn  you mean that this validation here https://github.com/fidelity/textwiser/blob/20c411596b7ae897a3bca83045221b4939ea3dbe/textwiser/options.py#L252 

is NOT validating the kwargs that go deep into the model, yes? And we stay away from "validating" arguments for downstream libraries and only validate our own arguments. 

This makes sense, as we possibly cannot check for everyhing. 

THEN, fixing the tests as above sounds like an easy solve. 

@nagireddyakshay  can you keep a simple test with that zero argument, which fails, add a decorator to skip that test so it does not break any pipeline, and inside the test, add your comments from this PR (the links you share to doc2vec etc..) Basically just narrate what we discussed here and what's happening here in that test for reference in future and explain our design decisions that, we know this fails but not asserting on purpose because it's in the downstream library etc.

Reply:
Done. will add a test and update the PR. Thank you @skadio.